frontend/src/metabase/components/TokenField/TokenField.tsx

```diff
@@ -455,18 +455,12 @@ export default class TokenField extends Component<
     } else {
       onChange(valueToAdd.slice(0, 1));
     }
-    // reset the input value
-    // setTimeout(() =>
-    //   this.setInputValue("")
-    // )
```

Comment:
these have been commented for 4 years. I figured we could probably remove them safely now.